modules/programs/sbt.nix

```
@@ -0,0 +1,122 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  renderPlugin = plugin: ''
+    addSbtPlugin("${plugin.org}" % "${plugin.artifact}" % "${plugin.version}")
+  '';
+
+  renderCredential = cred: ''
+    credentials += Credentials("${cred.realm}", "${cred.host}", "${cred.user}", "${cred.password}")
+  '';
+
+  sbtTypes = {
+    plugin = types.submodule {
+      options = {
+        org = mkOption {
+          type = types.str;
+          description = "The organization the artifact is published under";
```

Option descriptions should form complete sentences,
```suggestion
          description = "The organization the artifact is published under.";
```